### PR TITLE
add-getAllShops-service-controller

### PR DESCRIPTION
**Pull request by bavanya**

### CODE REVIEW

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetAllShopsResponse.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import com.google.gpay.virtualqueue.backendservice.model.Shop;
+
+import lombok.Data;
+import lombok.Setter;
+
+@Data
+@Setter
+public class GetAllShopsResponse {
+    private List<Shop> shopList = new ArrayList<>();
+}
```
Comment (stringtheory1234) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetAllShopsResponse.java:15]: add a new line here.

Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetAllShopsResponse.java]: yes Done

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetAllShopsResponse.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import com.google.gpay.virtualqueue.backendservice.model.Shop;
+
+import lombok.Data;
+import lombok.Setter;
+
+@Data
+@Setter
+public class GetAllShopsResponse {
+    private List<Shop> shopList = new ArrayList<>();
```
Comment (priytanvani) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetAllShopsResponse.java:30]: We should not be initializing the proto classes.

Comment (bavanya) [reply backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetAllShopsResponse.java]: Removed the initialization

File: backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VirtualQueueController.java [hunk 3]
```diff
@@ -36,14 +33,14 @@
...
     @Autowired
     public VirtualQueueService virtualQueueService;
-    
+
     @PostMapping("/shop")
     public CreateShopResponse addShop(@RequestBody CreateShopRequest createShopRequest) {
         return virtualQueueService.createShop(createShopRequest);
     }
 
-    @GetMapping("/token/{shopId}")
```
Comment (sdumka) [inline backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/controller/VirtualQueueController.java:45]: Can you ensure that this change has the PR merged by Ishita: https://github.com/googleinterns/gpay-virtual-queue/pull/19/files

**Pull request closed without merging**